src/NUnitEngine/nunit.engine/nunit.engine.csproj

```diff
@@ -64,6 +64,7 @@
       <Link>Agents\AgentExitCodes.cs</Link>
     </Compile>
     <Compile Include="Agents\RemoteTestAgentProxy.cs" />
+    <Compile Include="Internal\RuntimeFrameworks\DotNetFrameworkLocator.cs" />
```

Comment:
I was actually going to say that we should move this down to the other Internal files, but when @rprouse merges in the new csproj format the includes are going to disappear. So let us just keep it as it is.